com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/model/SubprocessPhase.java

```diff
@@ -0,0 +1,6 @@
+package com.sap.cloud.lm.sl.cf.core.model;
+
+public enum SubprocessPhase {
```

Comment:
I do not get why we need this.
If we need the blue_green phase, why not directly put it here - SubprocessPhase -> BlueGreenPhase
and the corresponding phases will be IDLE/LIVE

Reply:
Or can we use only the Phase.AFTER_RESUME in order to determine whether we are in IDLE or LIVE phases?